src/App.js

```diff
@@ -20,6 +21,8 @@ function App() {
         <Route path="/aboutus" />
         <Route path="/signin" />
         {menDisp ? <Menu /> : <Navbar />}
+        <Route path="/foodpage" component={FoodPage} />
```

Comment:
the path shouldn't include the word `page`, I specifically said the path should be `foods`, please make sure you read my comments well before making any changes